Scripts/GetEWSFolder/GetEWSFolder.py

```diff
@@ -0,0 +1,33 @@
+from CommonServerPython import *
+
+
+def convert_mail_to_json(item, folder):
```

Comment:
It's better to split here the JSON for subject, body, html so we will have all the information. We might expand it in the future and use more information